src/main/java/net/snowflake/ingest/utils/ParameterProvider.java

```diff
@@ -152,6 +142,14 @@ private void setParameterMap(
               BUFFER_FLUSH_INTERVAL_IN_MILLIS, MAX_CLIENT_LAG));
     }
 
+    /* STREAMING_ICEBERG should be the first thing to set as it affects other parameters */
```

Comment:
nit: ENABLE_ICEBERG_STREAMING should be ..

Reply:
Modified